app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -2201,6 +2206,8 @@ class BrowserTabFragment :
     @SuppressLint("SetJavaScriptEnabled")
     private fun configureWebView() {
         viewModel.configureBrowserBackground()
+        binding.browserContentLayout.layoutTransition.enableTransitionType(LayoutTransition.CHANGING)
```

Comment:
_Context:_ This was added to allow native transitions when new dax dialogs show/hide